packages/core/lib/debug/external.js

```diff
@@ -45,27 +33,32 @@ class DebugExternalHandler {
         }
       }
     } else {
-      sortedFetchers = allFetchers;
+      sortedFetchers = Fetchers;
     }
-    //to get the final list, we'll filter out ones that don't support this
+    const networkId = await new Web3(this.config.provider).eth.net.getId(); //note: this is a number
```

Comment:
BTW you don't need to do this. This is available on the config:
```suggestion
    const networkId = this.config.network_id; //note: this is a number
```

Reply:
Oh whoa. We've always been able to do this...?

Reply:
Yeah, found it yesterday!